internal/auth0/action.go

```diff
@@ -28,10 +28,10 @@ type ActionAPI interface {
 	// See: https://auth0.com/docs/api/management/v2#!/Actions/get_actions
 	List(opts ...management.RequestOption) (c *management.ActionList, err error)
 
-	// ListTriggers available.
+	// Triggers available.
 	//
 	// https://auth0.com/docs/api/management/v2/#!/Actions/get_triggers
-	ListTriggers(opts ...management.RequestOption) (l *management.ActionTriggerList, err error)
+	Triggers(opts ...management.RequestOption) (l *management.ActionTriggerList, err error)
```

Comment:
The `ListTriggers` has been deprecated in favor of `Triggers`. The func signature remains the same.